docs/getting-started/security-ui.asciidoc

```diff
@@ -182,6 +182,8 @@ Expand this section to access and manage additional security features:
 * <<event-filters, *Event filters*>>: View and manage event filters, which allow you to filter endpoint events you don't need to want stored in {es}.
 * <<host-isolation-exceptions, *Host isolation exceptions*>>: View and manage host isolation exceptions, which specify IP addresses that can communicate with your hosts even when those hosts are blocked from your network.
 * <<blocklist, *Blocklist*>>: View and manage the blocklist, which allows you to prevent specified applications from running on hosts, extending the list of processes that {elastic-defend} considers malicious.
+* <<response-actions-history, **Response actions history**>>: Find the history of response actions performed on hosts.
```

Comment:
```suggestion
* <<response-actions-history, **Response actions history**>>: View the history of response actions performed on hosts.
```

Reply:
Thanks @benironside! This was an edit based on @nastasha-solomon's suggestion [here](https://github.com/elastic/security-docs/pull/3861#pullrequestreview-1611614952), to avoid ableist language. Do you think 'examine' would sound better?

Reply:
@natasha-moore-elastic `Find` is OK in this context (yes, although super hard sometimes, we have to keep ableist language in mind). You can merge/backport this PR.